src/Microsoft.ML.Data/Transforms/ConcatTransform.cs

```diff
@@ -417,9 +417,9 @@ protected override void GetMetadataCore<TValue>(string kind, int iinfo, ref TVal
                 }
             }
 
-            private void GetCategoricalSlotRanges(int iiinfo, ref VBuffer<DvInt4> dst)
+            private void GetCategoricalSlotRanges(int iiinfo, ref VBuffer<Int32> dst)
```

Comment:
>Int32 [](start = 74, length = 5)

Please just use the keywords for types (so, in this case `int`), here and everywhere.

Reply:
I made this change across the code base using find-replace.